build/azure-pipelines.yml

```diff
@@ -0,0 +1,140 @@
+#############################################################################
+## ASP.NET Core                                                            ##
+## Build and test ASP.NET Core projects targeting .NET Core.               ##
+## Runs tests, creates NuGet packages:                                     ##
+## https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core ##
+#############################################################################
+
+# Variables & their default values
+variables:
+  buildConfiguration: 'Release'
+
+stages:
+- stage: Linux
+  jobs:
+
+  - job: Unit_Tests
+    displayName: 'Unit Tests'
+    pool:
+      vmImage: 'ubuntu-latest'
+    steps:
+  
+    - task: UseDotNet@2
+      displayName: 'Use .Net Core sdk 3.1.100'
+      inputs:
+        version: 3.1.100
```

Comment:
That SDK version is out of date. Use 3.1.x to automatically get the latest.

Reply:
I'll update, thanks